tools/webinstall/install.sh

```diff
@@ -1153,6 +1160,85 @@ install_linux_manual() {
     _CLEANUP_VERSION=""
 }
 
+# check_environment checks the environment for the kraftkit group.
+# If the group already exists, it means that it owns all default files.
+# $1: the group name
+# Returns:
+# Code: 0 if group doesn't exist, 1 if it exists
+check_environment() {
+    _che_group="$1"
+
+    need_cmd "$GETENT"
+
+    if [ -z "$($GETENT group "$_che_group")" ]; then
+        return 0
+    fi
+
+    say_debug "group $_che_group already exists"
+    return 1
+}
+
+# create_environment creates the kraftkit group and sets the permissions for
+# the default files (/var/kraftkit).
+# $1: the group name
+# Returns:
+# Code: 0 on success, 1 on error
+create_environment() {
+    _cre_group="$1"
+
+    need_cmd "$CHGRP"
+    need_cmd "$CHMOD"
+    need_cmd "$GROUPADD"
+    need_cmd "$MKDIR"
+    need_cmd "$USERMOD"
+    need_cmd "$WHOAMI"
+
+    say "creating $_cre_group group"
+    do_cmd "$GROUPADD $_cre_group"
+
+    say_debug "detecting current user"
+    _cre_user="$($WHOAMI)"
+
+    say "adding $_cre_user to $_cre_group group"
+    do_cmd "$USERMOD -a -G $_cre_group $_cre_user"
+
+    say "creating default directories"
+    _cre_default_dirs="/var/kraftkit"
```

Comment:
Can you make this a global program variable, `RUNTIME_DIR`. We also need a way to set this automatically if it is changed in the `~/.config/kraftkit/config.yaml` file. But let's leave that latter part for another day (e.g. when we have something like `kraft config set ...`